README.md

```diff
@@ -0,0 +1,12 @@
+# ServerMOTD
+
+A module which factors out the Server MOTD functionality into a module.
```

Comment:
No need to indicate that the functionality is being extracted, just state what the module does - in the future the extra note won't matter :-)